controllers/etcd_controller.go

```diff
@@ -998,6 +1030,16 @@ func (r *EtcdReconciler) removeOperationAnnotation(etcd *druidv1alpha1.Etcd) err
 	return nil
 }
 
+func (r *EtcdReconciler) updateEtcdStatusAsNotReady(etcd *druidv1alpha1.Etcd) (*druidv1alpha1.Etcd, error) {
+	etcdCopy := etcd.DeepCopy()
+	etcdCopy.Status.Ready = false
```

Comment:
Should `Ready` be a `*bool` instead of `bool` so that you can set it to `nil` here? Setting it to `false` is a little weird - actually, the statefulset might be running fine and controllers listening on the `.status.ready` field might misinterpret this temporary `ready=false` status. WDYM?